config/_default/menus.en.toml

```diff
@@ -78,4 +78,3 @@ url = "terms-conditions"
 
 [[footer_right]]
 name = "Contact & Acknowledgement"
-url = "contact"
```

Comment:
This line should not be removed. Otherwise, we lose access to the contact section.

Reply:
Solved!